### PR TITLE
Add REBUILD.changelog

### PR DESCRIPTION
**Pull request by Tal-or**
REBUILD.changelog is a top level file which we can use to trigger builds.

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,3 +83,4 @@ undeploy:
 gen-manifests:
 	@curl -L $(TOPOLOGYAPI_MANIFESTS)/crd.yaml
 	@hack/get-manifest-ds.sh
+
```
Comment (fromanirh) [inline Makefile:86]: next time I'd create a `REBUILD.changelog` top level file which we can use to trigger builds.

Comment (Tal-or) [reply Makefile:86]: Every day we learn something new :)

Comment (fromanirh) [reply Makefile:86]: Sure, just please note this is only a convention of ours. Such a file (neither the name or the content) has no special meaning for git, github, the CI or anything.

Comment (Tal-or) [reply Makefile:86]: Yes, I understood that. It's like ```README``` file convention.
Anyway, I took the opportunity of this PR to implement that.

**Pull request merged**